mfr/extensions/zip/templates/viewer.mako

```diff
@@ -0,0 +1,19 @@
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700" rel="stylesheet" type="text/css">
+<link rel="stylesheet" href="static/css/bootstrap.min.css">
+<link rel="stylesheet" href="static/css/default.css">
+<div style="word-wrap: break-word;" class="mfrViewer">
+    <code>
```

Comment:
`<code>` is a phrasing element like `<strong>` or `<em>` and shouldn't contain block-level elements, like `<h1>` and `<p>`.  In my local env, this causes weird floating elements before and after the code.  If you want the content to be monospaced and have a background, you should do that with CSS.

Reply:
Example:

![screen shot 2017-11-07 at 12 15 13 pm](https://user-images.githubusercontent.com/86835/32507461-7786c642-c3b5-11e7-9e0a-4b852dcab52c.png)